src/test/java/org/cactoos/io/TeeInputStreamTest.java

```diff
@@ -76,11 +76,11 @@ public void leftInputClosed() {
                     new OutputTo(write)
                 )
             ).intValue();
-            MatcherAssert.assertThat(
+            new Assertion<>(
                 "Can't use output after usage from TeeInput",
```

Comment:
@marceloamadeu please change here to "Must..." too

Reply:
@victornoel Done!